docker/nginx/conf.d/include/location-skylink

```diff
@@ -11,99 +11,37 @@ set_by_lua_block $skylink { return require("skynet.skylink").parse(ngx.var.skyli
 
 # $skylink_v1 and $skylink_v2 variables default to the same value but in case the requested skylink was:
 # a) skylink v1 - it would not matter, no additional logic is executed
-# b) skylink v2 - in a lua block below we will resolve the skylink v2 into skylink v1 and update 
-#      $skylink_v1 variable so then the proxy request to skyd can be cached in nginx (proxy_cache_key 
+# b) skylink v2 - in a lua block below we will resolve the skylink v2 into skylink v1 and update
+#      $skylink_v1 variable so then the proxy request to skyd can be cached in nginx (proxy_cache_key
 #      in proxy-cache-downloads includes $skylink_v1 as a part of the cache key)
 set $skylink_v1 $skylink;
 set $skylink_v2 $skylink;
 
-# variable for Skynet-Proof header that we need to inject 
+# variable for Skynet-Proof header that we need to inject
 # into a response if the request was for skylink v2
 set $skynet_proof '';
 
 # default download rate to unlimited
 set $limit_rate 0;
 
 access_by_lua_block {
-    -- the block below only makes sense if we are using nginx cache
-    if not ngx.var.skyd_disk_cache_enabled then
```

Comment:
all this code here and in another block that are protected with `ngx.var.skyd_disk_cache_enabled` shouldn't really matter because nginx doesn't access it when we have skyd cache enabled unless there is something wrong with this flag? I'd like to bring back this code to be able to limit this pull request to changes that fix the issue.

I just checked and `skyd_disk_cache_enabled` is expected to have either `"1"` or `"0"` so the check for `not ...` is definitely invalid here. It should be `if ngx.var.skyd_disk_cache_enabled == "0" then`.

reference https://github.com/SkynetLabs/skynet-webportal/pull/1762
```lua
# map skyd env variable value to "1" for true and "0" for false (expected by proxy_no_cache)
set_by_lua_block $skyd_disk_cache_enabled {
    return os.getenv("SKYD_DISK_CACHE_ENABLED") == "true" and "1" or "0"
}
```